glusterd2/volume/volume-utils.go

```diff
@@ -17,6 +17,21 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// GetRedundancy calculates redundancy count based on disperse count
+func GetRedundancy(disperse uint) int {
+	var temp, l, mask uint
+	temp = disperse
+	l = 0
```

Comment:
no need to set it explicitly to zero

Reply:
This func is moved from `volume-create-disperse.go` to utils. Implementation not changed

Reply:
fixed.